src/markdown-pages/install-with-kurl/removing-minio.md

```diff
@@ -0,0 +1,71 @@
+---
+path: "/docs/install-with-kurl/removing-minio"
+date: "2021-12-17"
+weight: 21
+linktitle: "Removing Minio"
+title: "Removing Minio from your Cluster"
+---
+
+As of [v2021.12.20-0](https://kurl.sh/release-notes/v2021.12.20-0), kURL clusters can be installed without addons having a peer dependency on object storage. 
+There is also a migration path for clusters to remove object storage, such as is the case with removing Minio.
```

Comment:
```suggestion
There is also a migration path for clusters to remove object storage, similar to removing Minio.
```

Reply:
Was going for something different here. I think I wanted to say removing Minio is a specific case of removing object storage. Suggestions on better wording appreciated.

Reply:
What about this?
```suggestion
There is also a migration path for clusters to remove object storage, whether Rook or MinIO is used for object storage.
```

Reply:
I like this suggestion if that fits what you were trying to achieve Dan.